lua/acf/shared/armor/rha.lua

```diff
@@ -4,11 +4,14 @@ local Material		= {}
 Material.id			= "RHA"
 Material.name		= "Rolled homogeneous Armor"
 Material.sname		= "RHA"
-Material.desc		= "Material which has no special traits. Your standart ACF armor."
+Material.desc		= "Simple, generic, but trusty steel. The standard armor everything else is compared to."
 Material.year		= 1900 -- Dont blame about this, ik that RHA has existed before this year but it would be cool to see: when?
 
 Material.massMod		= 1
-Material.curve		= 1
+Material.curve		= 0.99 --Slight and almost unnoticable penalty to high thickness armor
```

Comment:
I woudn't really change it. It is very insignificant and would make it confusing for other people